server/src/repositories/config.repository.ts

```diff
@@ -168,11 +173,11 @@ const getEnv = (): EnvData => {
     telemetry: {
       apiPort: Number(process.env.IMMICH_API_METRICS_PORT || '') || 8081,
       microservicesPort: Number(process.env.IMMICH_MICROSERVICES_METRICS_PORT || '') || 8082,
-      enabled: telemetryEnabled,
-      hostMetrics,
-      apiMetrics,
-      repoMetrics,
-      jobMetrics,
+      enabled: telemetries.size > 0,
+      hostMetrics: telemetries.has(ImmichTelemetries.HOST),
+      apiMetrics: telemetries.has(ImmichTelemetries.API),
+      repoMetrics: telemetries.has(ImmichTelemetries.REPO),
+      jobMetrics: telemetries.has(ImmichTelemetries.JOB),
```

Comment:
for workers, we just passed it in as `{ workers }`. Maybe we should do the same for telemetries as well. The services that use this can change their usage from `if(telemetry.hostMetrics)` to `if(telementry.has(ImmichTelemetry.HOST))` and `if(telemetry.size > 0)`, etc. I think that would be nice and consistent with workers.